utils/users.js

```diff
@@ -93,10 +93,18 @@ function getLowestLevelSkill(skills) {
   return { skill, level };
 }
 
+const addSearchDbQuery = (dbQuery, requestQuery) => {
+  return dbQuery
+    .orderBy("username")
+    .startAt(requestQuery.search.toLowerCase().trim())
+    .endAt(requestQuery.search.toLowerCase().trim() + "\uf8ff");
+};
```

Comment:
NIT: I don't think we needed to extract this

Reply:
moved this seperately to make `model` less cumbersome